docs/UserGuide.md

```diff
@@ -118,12 +96,10 @@ Format: `exit`
 
 ## Command summary
 
-| Action     | Format, Examples                                                                                                                               |
-|------------|------------------------------------------------------------------------------------------------------------------------------------------------|
-| **Add**    | `add n/NAME_OF_PET o/OWNER_NAME p/PHONE_NUMBER a/ADDRESS` <br> e.g., `add n/Peepee o/Peter p/98648252 a/13 Computing Drive, Singapore 117417`  |
-| **Delete** | `delete id` <br> e.g., `delete 3` (where 3 is the id of the pet in the system)                                                                 |
-| **Find**   | `find nameOfPet `<br> e.g., `find PeePee` (returns id of all pets called PeePee)                                                               |
-| **Time**   | `time id `<br> e.g.,`pickup 3 0900 1200` (where 3 is the id of the pet in the system)                                                          |
-| **Food**   | `food id `<br> e.g., `food 3` (where 3 is the id of the pet in the system)                                                                     |
-| **Owner**  | `owner o/OWNER_NAME` <br> e.g., `owner o/Jeff Lin`                                                                                             |
-| **Exit**   | `exit`                                                                                                                                         |
+| Action     | Format, Examples                                                                                                                              |
+|------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| **Add**    | `add n/NAME_OF_PET o/OWNER_NAME p/PHONE_NUMBER a/ADDRESS` <br> e.g., `add n/Peepee o/Peter p/98648252 a/13 Computing Drive, Singapore 117417` |
```

Comment:
Edits look good. Possible to add the diet command here as well: 
` diet id d/remark`